python/test/integration/test_quantile.py

```diff
@@ -9,13 +9,28 @@ def test_quantile_score_candidates():
     input_domain = dp.vector_domain(dp.atom_domain(T=int))
     input_metric = dp.symmetric_distance()
     candidates = [20, 33, 40, 50, 72, 100]
-    quant_trans = dp.t.make_quantile_score_candidates(input_domain, input_metric, candidates, alpha=0.5)
+    quant_trans = dp.t.make_quantile_score_candidates(
+        input_domain, input_metric, candidates, alpha=0.5
+    )
 
     assert quant_trans(list(range(100))) == [59, 33, 19, 1, 45, 100]
 
-    expo_meas = dp.m.then_report_noisy_max_gumbel(1., "min")
+    expo_meas = dp.m.then_report_noisy_max_gumbel(1.0, "min")
 
     quantile_meas = quant_trans >> expo_meas
     idx = quantile_meas(list(range(100)))
     assert candidates[idx] == 50
     assert quantile_meas.map(1) == 2.0
+
+
+def test_private_quantile():
+    input_space = dp.vector_domain(dp.atom_domain(T=int)), dp.symmetric_distance()
+    
+    m_median = input_space >> dp.m.then_private_quantile(
+        candidates=[20, 33, 50, 72, 100],
```

Comment:
Slightly weird set of candidates. Unless there's something special about these, make it more regular?
```suggestion
        candidates=[0, 25, 50, 75, 100],
```